src/providers/wms/qgswmscapabilities.cpp

```diff
@@ -905,6 +905,17 @@ void QgsWmsCapabilities::parseLayer( QDomElement const & e, QgsWmsLayerProperty&
 
         parseStyle( e1, styleProperty );
 
+        for ( int i = 0; i < layerProperty.style.size(); ++i )
+        {
+          if ( layerProperty.style[i].name == styleProperty.name )
```

Comment:
@m-kuhn: Look at the implementation of those two methods:
```c++
template <typename T>
inline const T &QList<T>::at(int i) const
{
  Q_ASSERT_X(i >= 0 && i < p.size(), "QList<T>::at", "index out of range");
  return reinterpret_cast<Node *>(p.at(i))->t();
}

template <typename T>
inline const T &QList<T>::operator[](int i) const
{
  Q_ASSERT_X(i >= 0 && i < p.size(), "QList<T>::operator[]", "index out of range");
  return reinterpret_cast<Node *>(p.at(i))->t();
}
```

I don't see why one would be faster than the other one.